designer/client/src/components/MenuBar/menu.tsx

```diff
@@ -18,6 +18,9 @@ const PlainButton = styled("button")({
     outline: "unset",
     padding: "unset",
     margin: "unset",
+    "&, &:focus": {
```

Comment:
what does the `&` part before `&:focus` do here? is it just the same as the current tag equivalent to `outline: "unset"`. If so, `outline: "unset"` 3 lines above can be removed.

Reply:
I removed "&",  ✔️